tcollector.py

```diff
@@ -956,6 +1003,15 @@ def parse_cmdline(argv):
                       help='Password to use for HTTP Basic Auth when sending the data via HTTP')
     parser.add_option('--ssl', dest='ssl', action='store_true', default=defaults['ssl'],
                       help='Enable SSL - used in conjunction with http')
+    parser.add_option('--listen-interface', dest='listen_interface', action='store',
```

Comment:
Do you think it would be good to make these "api-interface" and "api-port" instead? I believe it would more clearly communicate that these are not the ports used by the bridge collectors.

Reply:
`monitoring-interface`/`monitoring-port` maybe?